docs/content/downloads/releasenotes/release-0.7.0.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "0.7.0"
+menu:
+  main:
+    parent: "ReleaseNotes"
+    weight: 70
+---
+
+* [Release log](https://projects.eclipse.org/projects/modeling.elk/releases/0.7.0)
+* [Documentation](https://download.eclipse.org/elk/updates/releases/0.6.1/elk-0.7.0-docs.zip)
+* [Update site](https://download.eclipse.org/elk/updates/releases/0.7.0/)
+* [Zipped update site](https://download.eclipse.org/elk/updates/releases/0.7.0/elk-0.7.0.zip) (for offline use)
+* [Maven central](https://repo.maven.apache.org/maven2/org/eclipse/elk/) (for building pure Java projects that use ELK)
+
+
+
+## Details
+
+This is a major release which comes with quite a number of changes. Some of those are breaking changes, either in the usual API-breaking sense or in the sense that default layouts might look different. Those [issues](https://github.com/eclipse/elk/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Release+0.7.0%22+label%3Abreaking) and [pull requests](https://github.com/eclipse/elk/pulls?q=is%3Apr+is%3Aclosed+milestone%3A%22Release+0.7.0%22+label%3Abreaking) are now labeled with "breaking" to make such changes easier to spot.
+
+Here's a list of the most noteworthy changes. Head over to GitHub for [the full list](https://github.com/eclipse/elk/milestone/12?closed=1).
+
+
+### New Features and Enhancements
+
+* [#362](https://github.com/eclipse/elk/issues/362), [#613](https://github.com/eclipse/elk/pull/613): Most importantly, we finally have a new logo which appears on all of our websites. Joy!
+* [#105](https://github.com/eclipse/elk/issues/105), [#598](https://github.com/eclipse/elk/pull/598): Configuring spacings properly is one of the most complex things to get right. We thus added [a utility class](https://github.com/eclipse/elk/blob/master/plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/util/ElkSpacings.java) to help ease the pain a little.
+* [#533](https://github.com/eclipse/elk/issues/533), [#535](https://github.com/eclipse/elk/pull/535): _ELK Box_ and _ELK Rectangle Packing_ now support [content alignment](https://www.eclipse.org/elk/reference/options/org-eclipse-elk-contentalignment.html).
+* [#593](https://github.com/eclipse/elk/pull/593): _ELK Rectangle Packing_ now supports a configurable target width that the algorithm will try to achieve.
+* [#344](https://github.com/eclipse/elk/issues/344): ELK's JSON support nur supports a more relaxed JSON style.
+
```

Comment:
Maybe we should also add `considerModelOrder` for the `layered` algorithm here.

Reply:
Please do, you developed that feature. 🙂